### PR TITLE
Link to JIRA and better PR process

### PR DESCRIPTION
**Pull request by stevegore**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,29 +13,22 @@ This project is released with a [Code of Conduct](CODE_OF_CONDUCT.md). By contri
...
 
-1. In the repository navigation, click "Issues"
-1. Click "New issue"
-1. Choose "Bug Report" to create a report to help us improve or "Feature Request" to suggest an idea for the project
-1. Use the provided template to create your issue
-
-### More Information
-
-- [Creating an issue](https://help.github.com/en/articles/creating-an-issue)
+Please submit all issues to our [JIRA instance](https://q-ctrl.atlassian.net/secure/CreateIssue.jspa)
```
Comment (robertlove) [inline CONTRIBUTING.md]: @stevegore this is a public repo used by outside contributors so we can't link to Jira

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,29 +13,22 @@ This project is released with a [Code of Conduct](CODE_OF_CONDUCT.md). By contri
...
-1. Push to your fork and submit a pull request
+1. Push to your branch and submit a pull request
+1. Add reviewers
 1. Pat yourself on the back and wait for your pull request to be reviewed and merged
 
 Here are a few things you can do that will increase the likelihood of your pull request being accepted:
 
 - Follow our [Coding Standards](#coding-standards)
-- Spell variable names out in full using American English spelling (e.g. `optimized_pulse`, **NOT** `op`)
+- Spell variable names out in full using Australian English spelling (e.g. `optimized_pulse`, **NOT** `op`)
```
Comment (robertlove) [inline CONTRIBUTING.md]: @stevegore we use American English throughout

File: CONTRIBUTING.md (hunk 1 above)
Comment (stevegore) [reply CONTRIBUTING.md]: Fair enough.

File: CONTRIBUTING.md (hunk 2 above)
Comment (stevegore) [reply CONTRIBUTING.md]: Ok understood.

**Comment by stevegore** [comment]
I'm going to delete the branch and PR - misunderstood the intention of this.

**Pull request closed without merging**